src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/OrderingHelper.cs

```diff
@@ -246,12 +211,22 @@ private static IEnumerable<string> GetEvaluatedIncludes(IProjectTree projectTree
 
                 if (tree is IProjectItemTree2 tree2 && IsValidDisplayOrder(tree2.DisplayOrder))
                 {
+                    var include = tree2.Item.ItemName;
                     // Technically it is possible to have more than one of the same item names.
                     // We only want to add one of them.
                     // Sanity check
-                    if (hashSet.Add(tree2.Item.ItemName))
+                    if (hashSet.Add(include))
                     {
-                        includes.Add(tree2.DisplayOrder, tree2.Item.ItemName);
+                        // GetItemsByEvaluatedInclude is efficient and uses a MultiDictionary underneath.
+                        //     It uses this: new MultiDictionary<string, ProjectItem>(StringComparer.OrdinalIgnoreCase);
+                        var item = project.GetItemsByEvaluatedInclude(include).FirstOrDefault();
+
+                        // We only care about adding one item associated with the evaluated include.
```

Comment:
Is this true?  What if the include is a glob?

Reply:
I'm operating under the assumption in this case that we don't handle globs.

Reply:
F# projects will have globs, such as the `<Content/>` one from web projects.